npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
> This includes the one in npm.js,

what do you mean by this?

when does these packages get crawled and when does these packages goes though the jsonTransform method? whats the benefit of run these packages though the jsonTransform method? when does they be written out? 
i do not understand the workflow.